site/topics/parameters/comparing.rst

```diff
@@ -6,6 +6,76 @@ What Comparing Distributions *Means*
 Random Variables
 ================
 
+.. warning::
+
+    The content within this topic is kept at a high level and deliberately skips some formalisms and rigor. For the
+    purpose of this course, the level of detail provided should be sufficient.
+
+
+* Consider a random variable as some phenomenon that can be observed
+* When observing the random variable, some quantitative measurement can be taken
+
+    * The time it takes to recover from the flu
+    * The amount of nuts a squirrel buries a day
+    * The number of people with blond hair that walk by this building between 10am -- 11am on Tuesdays
+    * The best fitness value of a genetic algorithm run
+
+
+* Whatever the measurement is, they can be recorded and represented as distributions
+* Consider the following two random variables
+
+.. figure:: random_variable_1.png
+    :width: 666 px
+    :align: center
+
+    Output of some random variable represented as a box. The data is shown as a list of numbers and as a distribution.
+    A total of 50 data points were observed and recorded.
+
+
+.. figure:: random_variable_2.png
+    :width: 666 px
+    :align: center
+
+    Output of another random variable represented as a box. The data is shown as a list of numbers and as a
+    distribution. A total of 50 data points were observed and recorded.
+
+
+* Regardless of what these random variables are, one may wonder, *are these random variables actually different*?
+
+    * If considering the flu recovery time
+
+        * The random variables may be recovery times of two different groups of people
+        * Does one group recover faster?
+
+
+    * With the squirrels, does one squirrel bury more nuts in a day than the other?
+    * Does one year have more blond people walking past the building on Tuesdays from 10am -- 11am?
+    * Does one evolutionary computation algorithm implementation perform better than another?
```

Comment:
Reading this and thinking about it anew, I kinda feel like statistical tests don't exactly apply. The processes aren't actually random, rather the behaviour is entirely known and defined by the parameters. 

But I am probably overthinking. These tests compare two distributions, which I guess we can still obtain from a deterministic process.